cmd/multi_cmd.go

```diff
@@ -37,5 +37,5 @@ func multi(config *conf.GlobalConfiguration) {
 
 	models.RunHooks(bgDB, logrus.WithField("component", "hooks"))
 
-	api.ListenAndServe(l)
+	logrus.WithError(api.ListenAndServe(l)).Fatal("Error listening")
```

Comment:
> Fatal is equivalent to Print() followed by a call to os.Exit(1).

I think that is wrong. The docs are silly. It means that this will never exit cleanly.